requirements.yml

```diff
@@ -0,0 +1,9 @@
+name: cwl_assembly
```

Comment:
Is there a reason cwltool is not installed here as well?

Reply:
It breaks conda... too many dependencies